build_tools/release_to_github.py

```diff
@@ -83,38 +83,37 @@ def release(config, tag_name, release_sha, s3_release):
         exit(1)
 
     release = None
-    response = github.get("%s/releases" % target_repo, config.github_token)
+    response = github.get("%s/releases/tags/%s" % (target_repo, tag_name), config.github_token)
     if response:
-        for r in response:
-            if r.get("tag_name") == tag_name:
-                release = r
-                break
+        release = response
+
+    data = {
+        "tag_name": tag_name,
+        "name": release_name,
+        "body": "Defold version %s channel=%s sha1=%s date='%s'" % (config.version, channel, release_sha, datetime.now().strftime("%Y-%m-%d %H:%M:%S")),
+        "draft": draft,
+        "prerelease": pre_release
+    }
```

Comment:
We want to _always_ update the message, so that it reflects the latest build date and sha1